packages/blade/src/components/Icons/_Svg/Svg/Svg.native.tsx

```diff
@@ -1,8 +1,13 @@
+import styled from 'styled-components/native';
 import type { ReactElement } from 'react';
 import { Svg as SvgNative } from 'react-native-svg';
 import type { SvgProps } from './types';
-import { getStyledProps, makeAccessible } from '~utils';
-import BaseBox from '~components/Box/BaseBox';
+import { makeAccessible, useStyledProps } from '~utils';
+
+const StyledSvg = styled(SvgNative)((props) => {
+  const styledPropsCSSObject = useStyledProps(props);
+  return styledPropsCSSObject;
```

Comment:
I don't know how to explain it bit it looks illegal 🙈🙈 (mostly because in general we're used to reading the value from hook on top of component then use it below).

Reply:
Yup this looks fine as it is, hooks should stay on top, easy to miss out otherwise with refactors in case a conditional logic comes in between.